gobblin-compaction/src/main/java/gobblin/compaction/event/CompactionSlaEventHelper.java

```diff
@@ -56,8 +56,7 @@ public static void setUpstreamTimeStamp(State state, long time) {
   }
 
   private static void setDatasetUrn(Dataset dataset) {
-    dataset.jobProps().setProp(SlaEventKeys.DATASET_URN_KEY,
-        new Path(dataset.jobProps().getProp(MRCompactor.COMPACTION_DEST_DIR), dataset.topic()).toString());
+    dataset.jobProps().setProp(SlaEventKeys.DATASET_URN_KEY, dataset.outputPath());
```

Comment:
Is this correct? Originally `DATASET_URN_KEY` is set to the topic folder, e.g., /data/tracking/PVE. But `dataset.outputPath()` includes the time bucket.